addons/sale/i18n/sale.pot

```diff
@@ -1518,15 +1517,15 @@ msgstr ""
 
 #. module: sale
 #. odoo-python
-#: code:addons/sale/wizard/sale_make_invoice_advance.py:0
+#: code:addons/sale/models/sale.order.line.py:0
 #, python-format
-msgid "Down Payment: %s (Draft)"
+msgid "Down Payment: %(date)s (Draft)"
 msgstr ""
 
 #. module: sale
 #. odoo-python
 #: code:addons/sale/models/sale_order.py:0
-#: code:addons/sale/wizard/sale_make_invoice_advance.py:0
+#: code:addons/sale/models/sale.order.line.py:0
```

Comment:
Where is this change coming from ? :thinking:
There is no `Down Payments` string in `sale.order.line`

Reply:
I was pattern matching from this saas-17.1 PR #158206. However I see that in that PR, `Down Payments` string was added into in `sale.order.line`. I reverted the change back to the wizard as `Down Payments` line section isn't created in `sale.order.line` in older versions.